src/block/block_ext.c

```diff
@@ -919,7 +949,7 @@ __wt_block_extlist_merge(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_EXTLIST *
      * way to determine if the checkpoint is live so we cannot assert the locking here.
      */
 
-    __wt_verbose_debug2(session, WT_VERB_BLOCK, "merging %s into %s", a->name, b->name);
+    __wt_verbose(session, WT_VERB_BLOCK, "merging %s into %s", a->name, b->name);
```

Comment:
Please keep the existing debug log levels. You can use the config `verbose=[block:4]` to see all levels or `block:N` to observe logs up to level N.

Reply:
I've been restored here.

In addition, in order to keep ext range logs coincident, I have made the ext log level range changes here, for example:

1.https://github.com/wiredtiger/wiredtiger/blob/develop/src/block/block_ext.c#L560-L572
2.https://github.com/wiredtiger/wiredtiger/blob/develop/src/block/block_ext.c#L1056-L1072

thanks.

Reply:
I see, I'm wondering if it would be better to change the first message you point out to level 2.